package.json

```diff
@@ -31,7 +31,8 @@
     "flow": "flow status",
     "flow-check": "flow check",
     "coverage": "./scripts/generate-coverage.sh",
-    "build": "./scripts/build.sh"
+    "build": "./scripts/build.sh",
+    "quick-build": "./scripts/quick-build.sh"
```

Comment:
Ah wonderful! I didn't know the purpose of watch.sh. I will remove quick-build.sh.